electroncash/servers.json

```
@@ -145,4 +145,9 @@
         "t": "50001",
         "version": "1.5.0"
     }
+    "fulcrum.jettscythe.xyz": {
```

Hmm no longer valid JSON. JSON is pretty strict...

```suggestion
    },
    "fulcrum.jettscythe.xyz": {
```